### PR TITLE
Document try_sum aggregate function

### PR DESCRIPTION
**Pull request by kasiafi**
Docs should be added when the feature is fully deployed: https://github.com/duneanalytics/arrakis-jobs/pull/3618

### CODE REVIEW

**Comment by kasiafi** [comment]
The respective change has been deployed to production.

File: docs/query/DuneSQL-reference/Functions-and-operators/aggregate.md [hunk 1]
```diff
@@ -244,6 +244,11 @@ Returns `n` values of `x` associated with the `n` smallest of all input values o
 
 Returns the sum of all input values.
 
+#### try_sum()
+**``try_sum(uint256)``** &#8594 uint256
```
Comment (belen-pruvost) [inline docs/query/DuneSQL-reference/Functions-and-operators/aggregate.md:248]: Fancy arrow :D

**Review by belen-pruvost — APPROVED** [review]

**Comment by belen-pruvost** [comment]
@0xBoxer / @kasiafi - will this still be added to the docs, perhaps on the new repo?

**Comment by 0xBoxer** [comment]
https://github.com/duneanalytics/dune-api-docs/pull/64

already done

**Comment by 0xBoxer** [comment]
sorry my bad, just commented in the linear ticket for this

**Pull request closed without merging**